src/pages/fieldManagement/settings/defaultFieldComponents/SpeciesEditor.jsx

```diff
@@ -54,9 +53,8 @@ export default function SpeciesEditor({
     if (searchResultsError) {
       setStillGeneratingDisplay(false);
       clearTimeout(timerRef.current);
```

Comment:
This can be removed also. `searchResultsError` can be removed from the first condition. That means the second condition can just become and `else` block.